api/v1beta1/tempest_types.go

```diff
@@ -78,6 +78,11 @@ type ExtraImagesType struct {
 	// ID that should be assigned to the newly created image
 	ID string `json:"ID"`
 
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// Timeout duration for an image to reach the active state after its creation
+	ImageCreationTimeout int64 `json:"imageCreationTimeout"`
```

Comment:
Can we please update also the test-operator job here so that it tests this parameter? [1] 
```
      cifmw_test_operator_tempest_extra_images:
        - URL: https://download.cirros-cloud.net/0.6.2/cirros-0.6.2-x86_64-disk.img
          name: cirros-0.6.2-test-operator
          imageCreationTimeout: 300
          flavor:
            name: cirros-0.6.2-test-operator-flavor
            RAM: 512
            disk: 20
            vcpus: 1
```

[1] https://github.com/openstack-k8s-operators/test-operator/blob/4344305621b4ea280192750a9aae0a577e6824c0/.zuul.yaml#L66C10-L66C18

Reply:
Done!